app/views/partials/_post.html.erb

```diff
@@ -3,14 +3,14 @@
     <h2><%= "#{post.title} by #{post.user.name}" %></h2>
   </div>
   <p class='post-text'>
-    <% if short && post.body.split(' ').length > 15 %>
-      <%= post.body.split(' ').slice(0,15).join(' ') %>...
+    <% if short && post.text.split(' ').length > 15 %>
+      <%= post.text.split(' ').slice(0,15).join(' ') %>...
     <% else %>
-      <%= post.body %>
+      <%= post.text %>
     <% end %>
```

Comment:
- [ ] Fantastic job incorporating the changes suggested by the previous reviewer! 🌟 However, during my testing of your application, I encountered a small hiccup related to recent modifications in your views.

  - Upon inspecting the columns in your `post` table, I observed that there's no column named `text`. Interestingly, in your views, you reference a column called `text` on the `post` table, which seems to be nonexistent.

  - To address this, I recommend replacing the column name `body` with `text` in your views. This adjustment aligns with the column names specified in the `ERD` diagram provided at the project's outset. See below for more details:
  
<img width="693" alt="Screenshot 2023-11-18 at 14 31 02" src="https://github.com/Nessrine88/Blog-App/assets/107192076/73c8607e-2e4f-4029-8dcb-d64412480405">


Let's ensure our code reflects the project's initial design accurately! 🚀 Happy coding! 👏